apps/web/src/modules/job/apply/pages/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { UserType } from '@modela/dtos'
+import {
+  Button,
+  CircularProgress,
+  Divider,
+  Link,
+  Typography,
+} from '@mui/material'
+import withGuard from 'common/hoc/withGuard'
+
+import SelectResume from '../components/SelectResume'
+import useResume from './hooks'
+import { CardContainer, RootContainer } from './styled'
+
+const JobApplyPage = () => {
+  const { jobId, jobTitle, handleSuccess, loading, resumes, Id, setId } =
+    useResume()
+  if (loading) return <CircularProgress />
+
+  return (
+    <RootContainer>
+      <CardContainer>
+        <Typography variant="h6" textAlign="center">
+          สมัครงาน
+        </Typography>
+        <Divider />
+        <Typography variant="body1">
+          {'งานที่ต้องการสมัคร: '}
+          <Link
+            variant="body1"
+            color="primary"
+            sx={{ cursor: 'pointer', textDecoration: 'none' }}
+            href={`/job/${jobId}`}
+          >
+            {jobTitle}
+          </Link>
+        </Typography>
+        <SelectResume resumes={resumes} Id={Id} setId={setId}></SelectResume>
```

Comment:
This should fix along with `Id`
```suggestion
        <SelectResume resumes={resumes} id={id} setId={setId}></SelectResume>
```